ios/LayoutReanimation/REASnapshot.m

```diff
@@ -79,6 +79,17 @@ - (void)makeSnapshotForView:(UIView *)view useAbsolutePositionOnly:(BOOL)useAbso
       // Identity matrix is an default value
       _values[@"transformMatrix"] = @[ @(1), @(0), @(0), @(0), @(1), @(0), @(0), @(0), @(1) ];
     }
+#if defined(RCT_NEW_ARCH_ENABLED) || defined(TARGET_OS_TV)
+    _values[@"borderRadius"] = @(0);
+#else
+    RCTView *viewController = (RCTView *)view;
```

Comment:
Why `viewController`? It's not a `UIViewController`

Reply:
Resolved in: https://github.com/software-mansion/react-native-reanimated/pull/4103/commits/2864be8d241a385596d3aa3df9ab02fa510a676d